Model/src/BoardPackage/PropertySquare.java

```diff
@@ -1,7 +1,261 @@
 package BoardPackage;
 
-/**
- * Created by unclear on 2/5/18.
- */
-public class PropertySquare {
+import GamePackage.Player;
+
+/********************************************************************
+ * The PropertySquare class is a BoardSquare class that defines the
+ * BoardSquares that are properties. It tracks variables related to
+ * properties such as the PRICE of purchase and the rent cost of
+ * landing on this BoardSquare.
+ *
+ * @author David Baas
+ * @version 2/12/2018
+ ********************************************************************/
+public class PropertySquare extends BoardSquare{
+
+    /**An int to hold the PRICE of the property*/
+    private final int PRICE;
+
+    /**An int to hold the property's mortgage value*/
+    private final int MORTGAGE_VAL;
+
+    /**An int to hold the cost of rent w/ no houses.*/
+    private final int BASE_RENT;
+
+    /**An int to hold the cost of rent w/ one house*/
+    private final int ONE_HOUSE_RENT;
+
+    /**An int to hold the cost of rent w/ two houses*/
+    private final int TWO_HOUSE_RENT;
+
+    /**An int to hold the cost of rent w/ three houses*/
+    private final int THREE_HOUSE_RENT;
+
+    /**An int to hold the cost of rent w/ four houses*/
+    private final int FOUR_HOUSE_RENT;
+
+    /**An int to hold the cost of rent w/ a hotel*/
+    private final int HOTEL_RENT;
+
+    /**An int to track how many houses are on the property*/
+    private int numHouses;
+
```

Comment:
I think we also need hasHotel (boolean) variable to keep track of if the square has a hotel. It was listed in the original issue.

Reply:
Added the hasHotel var.